packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -280,22 +288,43 @@ const ApiTokenCreateView = () => {
             return (
               <Form>
                 <FormHead
-                  apiToken={apiToken}
-                  setApiToken={setApiToken}
+                  backUrl="/settings/api-tokens"
+                  title={{
+                    id: 'Settings.apiTokens.createPage.title',
```

Comment:
we need to add this value `Settings.apiTokens.createPage.title` inside packages/core/admin/admin/src/translations/en.json

Reply:
added 'Settings.apiTokens.createPage.title'